src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -48,18 +50,15 @@ class CallActor(call: Call, locallyQualifiedInputs: Map[String, WdlValue], backe
   private def handleStart(): Unit = {
     // Record the original sender here and not in the yield over the `Future`, as the
     // value of sender() when that code executes may be different than what it is here.
-    log.info("In handleStart for " + call.fullyQualifiedName)
     val originalSender = sender()
     val backendInputs = backend.adjustInputPaths(call, locallyQualifiedInputs)
-    log.info("Got backend inputs for " + call.fullyQualifiedName + ": " + backendInputs)
 
     val tryCommand = for {
       commandLine <- call.instantiateCommandLine(backendInputs)
     } yield {
-      log.info("Command line for " + call.fullyQualifiedName + ": " + commandLine)
+      log.info(s"$tag: launching `$commandLine`")
```

Comment:
So I know you all roll your eyes at my obsession with small closures in these cases but it did lead to me taking longer than it should've to spot the issue w/ the original code when I was looking at it w/ @kcibul today - although granted that could have been due to looking at someone else's laptop screen. It took me a while to realize this closure was as long as it was so I was thinking the problem was something else entirely.

Also with the new code, I don't know that the for comp is worth the hassle here - isn't that just a `map`? If `commandLine` is a `Success`, do X, otherwise do Y.